### PR TITLE
Fix empty glob in boost.dynamic_bitset and .random

### PR DESCRIPTION
**Pull request by hofbi**
More boost fixes for empty globs. I am also adding `--incompatible_disallow_empty_glob` to the presubmit jobs to catch this in future versions.

Needs skip-url-stability-check

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @wep21, @Vertexwahn, modules you maintain (boost.dynamic_bitset, boost.random) have been updated in this PR. Please review the changes.

File: modules/boost.dynamic_bitset/1.83.0.bcr.1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - windows
+  bazel: [7.x]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+      - '--process_headers_in_dependencies'
+      - '--incompatible_disallow_empty_glob'
```
Comment (wep21) [inline modules/boost.dynamic_bitset/1.83.0.bcr.1/presubmit.yml:18]:
```suggestion
```
incompatible_disallow_empty_glob is default for rolling

File: modules/boost.dynamic_bitset/1.83.0.bcr.1/MODULE.bazel [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+bazel_dep(name = "boost.assert", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.config", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.container_hash", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.core", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.integer", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.move", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.static_assert", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.throw_exception", version = "1.83.0.bcr.1")
+bazel_dep(name = "rules_cc", version = "0.0.9")
```
Comment (wep21) [inline modules/boost.dynamic_bitset/1.83.0.bcr.1/MODULE.bazel:16]:
```suggestion
bazel_dep(name = "rules_cc", version = "0.0.13")
```

File: modules/boost.random/1.83.0.bcr.1/MODULE.bazel [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+bazel_dep(name = "boost.dynamic_bitset", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.integer", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.io", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.range", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.static_assert", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.system", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.throw_exception", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.type_traits", version = "1.83.0.bcr.1")
+bazel_dep(name = "boost.utility", version = "1.83.0.bcr.1")
+bazel_dep(name = "rules_cc", version = "0.0.9")
```
Comment (wep21) [inline modules/boost.random/1.83.0.bcr.1/MODULE.bazel:21]:
```suggestion
bazel_dep(name = "rules_cc", version = "0.0.13")
```

File: modules/boost.random/1.83.0.bcr.1/presubmit.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+matrix:
+  platform:
+    - debian10
+    - debian11
+    - macos
+    - macos_arm64
+    - ubuntu2004
+    - ubuntu2204
```
Comment (wep21) [inline modules/boost.random/1.83.0.bcr.1/presubmit.yml:8]:
```suggestion
    - ubuntu2204
    - ubuntu2404
```

File: modules/boost.random/1.83.0.bcr.1/presubmit.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    - windows
+  bazel: [7.x]
```
Comment (wep21) [inline modules/boost.random/1.83.0.bcr.1/presubmit.yml:10]:
```suggestion
  bazel: [7.x, rolling]
```

File: modules/boost.random/1.83.0.bcr.1/presubmit.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+      - '--process_headers_in_dependencies'
+      - '--incompatible_disallow_empty_glob'
```
Comment (wep21) [inline modules/boost.random/1.83.0.bcr.1/presubmit.yml:18]:
```suggestion
```

**Comment by hofbi** [comment]
@meteorcloudy could you add the skip-url-stability-check check label please

File: modules/boost.dynamic_bitset/1.83.0.bcr.1/source.json [hunk 7]
```diff
@@ -0,0 +1,10 @@
+{
+    "integrity": "sha256-WVdk7ci9FDJinetJ/4fnRSpuQ5MKL/+kJLJUPtzyrYA=",
+    "strip_prefix": "dynamic_bitset-boost-1.83.0",
+    "url": "https://github.com/boostorg/dynamic_bitset/archive/refs/tags/boost-1.83.0.tar.gz",
+    "patch_strip": 0,
+    "overlay": {
+        "MODULE.bazel": "sha256-AEjsJtf6t9krwldyRPucAM43csO8vxxVAkDeRb082IA=",
```
Comment (wep21) [inline modules/boost.dynamic_bitset/1.83.0.bcr.1/source.json:7]:
```suggestion
        "MODULE.bazel": "sha256-IJOp0JFZuXraZ+03BJL2HHda599QGMi6nGkFPteBPas=",
```

File: modules/boost.random/1.83.0.bcr.1/source.json [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{
+    "integrity": "sha256-5qVfaZ7Uy/d5CaYd+MBJGxqt5GxSrnYMDC3zdhkZrV8=",
+    "strip_prefix": "random-boost-1.83.0",
+    "url": "https://github.com/boostorg/random/archive/refs/tags/boost-1.83.0.tar.gz",
+    "patch_strip": 0,
+    "overlay": {
+        "MODULE.bazel": "sha256-eLsnymtusQ1JfxstPlwmWCi9VuJp9QRmwfBzzvKe7lI=",
```
Comment (wep21) [inline modules/boost.random/1.83.0.bcr.1/source.json:7]:
```suggestion
        "MODULE.bazel": "sha256-yr47qCDJWIqcoiVIuIrYxLMHvghWkbKG/w2uikayX6A=",
```

**Review by wep21 — APPROVED** [review]
LGTM

**Review by bazel-io — APPROVED** [review]
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

**Pull request merged**